examples/fedora/rpminspect.fmf

```diff
@@ -0,0 +1,44 @@
+/local:
+    summary:
+        Generic rpminspect analysis test on localhost.
+        The rpminspect tool takes care of downloading the artifact.
+    provision:
+        how: local
+    prepare:
+        how: shell
+        script: sudo dnf install -y rpminspect
+    execute:
+        how: shell
+        # Note: script MUST be passed to 'tmt run' on command line
+        # e.g. tmt run -a execute --script "rpminspect BUILD_NVR"
+
+/container:
+    summary:
+        Generic rpminspect comparison test in a prepared container with profile 'gating'.
+        The rpminspect tool takes care of downloading the artifacts.
+    provision:
+        how: container
+        image: rpminspect-container-name
+    execute:
+        how: shell
+        # Note: script MUST be passed to 'tmt run' on command line
+        # e.g. tmt run -a execute --script "rpminspect BEFORE_NVR AFTER_NVR"
+
+/container-with-download:
+    summary:
+        Generic rpminspect comparison test in a prepared container with profile 'gating'.
+        Uses download prepare step to download the artifacts.
+    provision:
+        how: container
+        image: rpminspect-container-name
+    prepare:
+        - how: download
+          artifact: https://koji.fedoraproject.org/koji/buildinfo?buildID=1453726
+          destination: /tmp/systemd-244.1-3.fc32
```

Comment:
I wonder whether this shouldn't be simply relative path (without the `tmp`), as in prepare we definitely need to [download the builds](https://github.com/psss/tmt/pull/159/files#diff-4476c8abdfef907fe037b9351ba16b5cR158-R159) before install anyway. So could we maybe make the downloaded path symlinked from where the script runs?